src/corehost/cli/deps_format.cpp

```diff
@@ -350,15 +352,15 @@ bool deps_json_t::load(bool portable, const pal::string_t& deps_path, const rid_
     // If file doesn't exist, then assume parsed.
     if (!pal::file_exists(deps_path))
     {
-        trace::verbose(_X("Deps file does not exist [%s]"), deps_path.c_str());
+        trace::verbose(_X("Deps.json file [%s] does not exist"), deps_path.c_str());
         return true;
     }
 
     // Somehow the file stream could not be opened. This is an error.
     pal::ifstream_t file(deps_path);
     if (!file.good())
     {
-        trace::error(_X("Could not open file stream on deps file [%s]"), deps_path.c_str());
+        trace::error(_X("Could not open file stream on deps.json file [%s]"), deps_path.c_str());
```

Comment:
Since this is an unknown I/O failure, lets say `Unknown error loading [deps_path].` (If there's any way to get an error code or something from the OS, it'd be good to print that out too.